content/administration/maintain/google_oauth.rst

```diff
@@ -0,0 +1,208 @@
+========================================
+Setup Google Gmail using OAuth with Odoo
+========================================
```

Comment:
Ah maybe this, though!
No need to add "with Odoo". Else, all titles would have it :D